Cartfile

```diff
@@ -0,0 +1,3 @@
+github "LoopKit/LoopKit" "loop-release/v2.2.5"
```

Comment:
Carthage is used if you want to run the tests without involving a Loop Workspace. This is not really something we need to do but the support is there. There are some complexities to making this continue to work with Apple Silicon devices, so this will need rethought if we actually want to support this in the long run.

Reply:
I'm running on an Apple Silicon device so we probably need to consider that. I've been tempted to try to convert everything to SPM but haven't had the time to learn it yet.

Reply:
To clarify  - you can still run these tests from the Loop Workspace -- the limitation is just for the standalone OmniBLE which I'm not sure anyone will use -- and is a limitation for existing Loop dependencies that also have unit tests.

My hesitation on using Swift Packages is I don't know if the swift community plans to use Swift Packages yet (i.e. Is Pete going to use that in an upcoming dev/master version?).  I'd suggest we defer SPM integration efforts for a separate initiative outside this PR. This PR is using the standard integration procedure at the moment.

Reply:
Sounds good. I'm open to a PR that moves CryptoSwift to a Carthage package but I didn't know how to do that.

Reply:
For adding dependencies to submodules, Pete suggests those dependencies be added as _other_  submodule. I had this same need in the Remote Bolus integration. Here is where it is added as a submodule  https://github.com/LoopKit/LoopWorkspace/pull/12/files

To clarify why Carthage is mentioned in this PR though -- That is just to build OmniBLE tests as as standalone test target without involving LoopWorkspace. To be honest though, I don't think there is a great strategy known at the moment to build as stand-alone tests for Loop integrations. Carthage works but is a bad solution IMO since it doesn't work with M1s -- and Loop is moving away from Carthage without a clear replacement.  

The Swift Package Manager solution would be best long term however I don't want to inject that until we know Loop is moving that direction -- which is why I'm just sticking with the Carthage method here for now.

Reply:
That's kinda where I am at the moment except I didn't feel like figuring out how to get it working with CyrptoSwift so it is an SPM package for now. We'll leave it for now and see where it ends up.